keras/ops/core.py

```diff
@@ -168,6 +168,14 @@ def compute_output_spec(self, inputs, start_indices, updates):
         return KerasTensor(inputs.shape, dtype=inputs.dtype)
 
 
+class Scan(Operation):
+    def call(f, init, xs, length, reverse, unroll):
+        return backend.core.scan(f, init, xs, length, reverse, unroll)
+
+    def compute_output_spec(self, f, init, xs, length, reverse, unroll):
+        return KerasTensor(xs.shape, dtype=xs.dtype)
```

Comment:
This doesn't look correct -- it should match the shape of the output tensors. To start with, there are 2 tensors in this case.

Reply:
okay I have added 2 tensors to the return statement  but I am not sure if they are correct